libqfieldsync/project_checker.py

```diff
@@ -289,6 +294,45 @@ def check_project_is_dirty(self) -> Optional[FeedbackResult]:
         else:
             return None
 
+    def check_for_conflicting_base_filenames(self) -> Optional[FeedbackResult]:
+        conflicting_files: List[Path] = []
+        try:
+            project_file_path = Path(self.project.fileName())
+
+            if not project_file_path.is_file():
+                return None
+
+            home_path: Path = project_file_path.parent
+            project_base_name: str = project_file_path.stem
+
+            if not home_path.is_dir():
+                return None
+
+            for item in home_path.rglob("*"):
+                if (
+                    item.is_file()
+                    and item != project_file_path
+                    and item.stem == project_base_name
+                    and not item.name.endswith("~")
+                    and ".qfieldsync" not in item.relative_to(home_path).parts
+                ):
+                    conflicting_files.append(item)
+
+        except (TypeError, Exception):
+            pass
+
+        if conflicting_files:
+            return FeedbackResult(
+                self.tr(
+                    'The project "{}" shares its base name with the following files, which might cause issues: {}.'
+                ).format(
+                    project_base_name,
+                    (", ".join([f'"{path}"' for path in conflicting_files])),
+                )
```

Comment:
```suggestion
                self.tr(
                    'The project "{}" shares its base file name ({}) with the following files, which might cause issues: {}.'
                ).format(
                    project_base_name,
                    project_file_path.name,
                    (", ".join([f'"{path}"' for path in conflicting_files])),
                )
```

A bit more verbose warning message.

Reply:
Great!.... Indeed much better!

Reply:
Great!..... Indeed much better!